compiler/surface/lexer_ro.cppo.ml

```diff
@@ -0,0 +1,167 @@
+(* This file is part of the Catala compiler, a specification language for tax and social benefits
+   computation rules. Copyright (C) 2020 Inria, contributors: Denis Merigoux
+   <denis.merigoux@inria.fr>, Emile Rolley <emile.rolley@tuta.io>
```

Comment:
Please set yourself as contributor :)

Reply:
I simply copied the Polish implementation, wasn't sure if to add or no and what. No problem if no mention, it's not a major contribution.

Reply:
it can be useful to have a contact if the file is in need of maintainance later on ; no problem if you stick to your Github handle of course.

Reply:
Should I leave 'Denis Merigoux <denis.merigoux@inria.fr>, Emile Rolley <emile.rolley@tuta.io>' and then add myself or just add myself as the one that is responsible for the Romanian parser? Sorry, the etiquette is not clear.